scenes/Cell.gd

```diff
@@ -0,0 +1,20 @@
+extends Area2D
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+func _draw():
+	var collision = get_node("CollisionShape2D").shape.get_rect()
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	pass
+
+
+func _on_input_event(viewport, event, shape_idx):
+	if event is InputEventMouseButton and event.button_index == MOUSE_BUTTON_LEFT and event.pressed:
+		print("assigning X")
```

Comment:
You want to leave these debug statements in?

Reply:
Yeah, I'm choosing to just leaving it there for the moment until we work on the game manager part to handle the logic to toggle it based on the current player.